gen/definitions/profile_parcels/service_multicast.yaml

```diff
@@ -0,0 +1,177 @@
+---
+name: Service Multicast
+rest_endpoint: /v1/feature-profile/sdwan/service/%v/routing/multicast
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_service_feature_profile.test.id
+  - model_name: sptOnly
+    data_path: [basic]
+    mandatory: true
+    example: false
+  - model_name: local
+    tf_name: local_replicator
+    data_path: [basic, localConfig]
+    mandatory: true
+    example: false
+  - model_name: threshold
+    data_path: [basic, localConfig]
+    example: 10
+  - model_name: interface
+    tf_name: igmp_interfaces
+    id: true
+    data_path: [igmp]
+    attributes:
+    - model_name: interfaceName
+      example: GigabitEthernet1
+    - model_name: version
+      example: 2
+    - model_name: joinGroup
+      tf_name: join_groups
+      attributes:
+      - model_name: groupAddress
+        example: 224.0.0.0
+      - model_name: sourceAddress
+        example: 1.2.3.4
+  - model_name: enableSSMFlag
+    tf_name: pim_enable_source_specific_multicast
+    data_path: [pim, ssm, ssmRangeConfig]
+    mandatory: true
+    example: true
+  - model_name: range
+    tf_name: pim_spt_threshold
```

Comment:
pim_source_specific_multicast_access_list

Reply:
The pim_spt_threshold should be a separate parameter under pim, ssm section but it seems it's missing in this yaml.